agent/service.go

```diff
@@ -375,6 +386,15 @@ func (ts *TaskService) Exec(requestCtx context.Context, req *taskAPI.ExecProcess
 	logger := log.G(requestCtx).WithField("TaskID", taskID).WithField("ExecID", execID)
 	logger.Debug("exec")
 
+	defer func() {
+		if err != nil {
+			cleanupErr := ts.doCleanup(taskID, execID)
+			if cleanupErr != nil {
+				logger.WithError(err).Error("failed to cleanup task")
```

Comment:
can we include `cleanupErr` as well ? like `failed to cleanup task: %v`

Reply:
Yep this was an oversight, will fix